src/database/models/BirthdayModel.ts

```diff
@@ -0,0 +1,8 @@
+import {model, Schema} from 'mongoose';
+
+export const Birthday = new Schema({
+  _id: String,
+  birthdate: Date,
+});
+
+export default model('Birthday', Birthday, 'Birthdays');
```

Comment:
I like the idea of using models to communicate stored database data

Reply:
Me too I think it's great.